src/System.Numerics.Tensors/System.Numerics.Tensors.sln

```diff
@@ -0,0 +1,28 @@
+﻿
```

Comment:
I prefer a single solution for my development since I only care about this library.